Correct CDP name (#2543)
MicrosoftDocs/edge-developer: microsoft-edge/webview2/samples/wv2cdpextensionwpfsample.md

1. If a newer release version of the **Microsoft.Web.WebView.DevToolsProtocolExtension** SDK is listed, click the **Update** button.

> <!-- For more information, see [WebView2 CDP Extension](https://aka.ms/webviewcdpnuget).  todo: 404, ie: defaults to bing -->

Author: @novac42 @nishitha-burman -- Where to link to?  At live page, 
https://learn.microsoft.com/en-us/microsoft-edge/webview2/samples/wv2cdpextensionwpfsample#step-8---update-the-webview2-cdp-extension
click "WebView2 CDP Extension" link at bottom of section, which tries to go to
https://aka.ms/webviewcdpnuget
but fails and falls back to Bing home page: 
https://www.bing.com/?ref=aka&shorturl=webviewcdpnuget

At NuGet site https://www.nuget.org is there a "WebView2 CDP extension" package?  
https://www.nuget.org/packages?q=cdp -- can't find "webview2"
https://www.nuget.org/packages?q=cdp+webview2 -- 0 hits

Author: I don't exactly think the so-called "WebView2 CDP extension" NuGet package has been deleted; I just think the link text (package name) & the aka link (which i had to comment out b/c 404) are both outdated.  So I'm closing this comment after adding a link to the real NuGet package in the See Also section:

https://www.nuget.org/packages/Microsoft.Web.WebView2.DevToolsProtocolExtension

Per article https://learn.microsoft.com/en-us/microsoft-edge/webview2/how-to/chromium-devtools-protocol

> For more information about the WebView2 CDP Extension, see [Use the Chrome DevTools Protocol (CDP) in WebView2 apps](../how-to/chromium-devtools-protocol.md)

Author: ```suggestion
See also:
* [Use the Chrome DevTools Protocol (CDP) in WebView2 apps](../how-to/chromium-devtools-protocol.md)
```
was missing period in old format

Author: fixed locally

> For more information about the WebView2 CDP Extension, see [Use the Chrome DevTools Protocol (CDP) in WebView2 apps](../how-to/chromium-devtools-protocol.md)

Author: ```suggestion
See also:
* [Use the Chrome DevTools Protocol (CDP) in WebView2 apps](../how-to/chromium-devtools-protocol.md)
```
was missing period in old format
